slither/printers/call/call_graph.py

```diff
@@ -227,11 +232,11 @@ def output(self, filename):
         with open(all_contracts_filename, "w", encoding="utf8") as f:
             info += f"Call Graph: {all_contracts_filename}\n"
 
-            # Avoid dupplicate funcitons due to different compilation unit
-            all_functionss = [
+            # Avoid duplicate functions due to different compilation unit
+            all_functions = [
```

Comment:
So actually the double `s` is here because if the same variable is used twice, it will have two different types (list of list of functions, versus a list of functions).

Using `ss` is a shortcut to have two variables without having a lot of different names and it is used in different parts of the codebase, but it's not actually documented